ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -4,11 +4,21 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
 cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
 export AWS_SHARED_CREDENTIALS_FILE=$cluster_profile/.awscred
 export AZURE_AUTH_LOCATION=$cluster_profile/osServicePrincipal.json
 
 echo "Deprovisioning cluster ..."
 cp -ar "${SHARED_DIR}" /tmp/installer
-openshift-install --dir /tmp/installer destroy cluster
-cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}/"
+openshift-install --dir /tmp/installer destroy cluster &
+
+set +e
+wait "$!"
+ret="$?"
+set -e
+
+if [[ -f /tmp/installer/.openshift_install.log ]]; then
+  cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}"
+fi
```

Comment:
when do we expect the log file to not exist?

Reply:
> when do we expect the log file to not exist?

From [the PR topic][1]:

> Only attempt to gather `.openshift_install.log` if it exists. This way we do not have a failed `cp` kicking us out of the script and blocking the execution of subsequent gathers.

And the guard is mostly because it was easy to do.  But yeah, I'm having a hard time thinking of a case where we'd have no log but *would* have a `kubeconfig` and `metadata.json`.  Do you want me to drop the guard?

[1]: https://github.com/openshift/release/pull/7936#issue-393871804

Reply:
similar to my comment https://github.com/openshift/release/pull/7936#issuecomment-604148224

I think we should maybe try to reduce this conditional logic where ever unnecessary.

Reply:
Dropped the conditional with 1044cf8d1 -> 1a7b314f9.